assembly/src/errors.rs

```diff
@@ -19,7 +19,7 @@ pub enum AssemblyError {
     DuplicateProcName(String, String),
     DuplicateProcId(ProcedureId),
     ExportedProcInProgram(String),
-    ImportedProcModuleNotFound(ProcedureId),
+    ImportedProcModuleNotFound(ProcedureName, ProcedureId),
```

Comment:
I think we should consider the possibility when procedure name is not available. This could happen because AST's could be serialized without import maps. So, I think we should change this variant to something like:
```Rust
ImportedProcModuleNotFound(ProcedureId, Option<ProcedureName>),
```